Sources/PackageLoading/ManifestLoader+Validation.swift

```diff
@@ -107,11 +107,6 @@ public struct ManifestValidator {
                 continue
             }
 
-            guard traitName.lowercased() != "default" && traitName.lowercased() != "defaults" else {
-                diagnostics.append(.defaultTraitName())
```

Comment:
Wouldn't we want to keep this error? A user could still declare both `.default` and a trait named `"defaults"` at the same time with unintended consequences, right?

Reply:
I intentionally removed it because a user declaring a `default` trait is just an alternative spelling to using `.default(enabledTraits:)`. I don't want to ban this since a user could add a different description if they wanted. With this change the `default` trait is just serialised as any other trait.
I also thought about keeping the ban on declaring a  `defaults` but decided that it is not worth the restriction since it poses no harm. I could add the ban for `defaults` back and just remove the `default` one but I don't fell strongly about it

Reply:
The spelling needs to be adjusted then. The helper function is called `.default`, while the trait is called `"defaults"`.  How's about keeping the function `.default` as it is now, but making the string spelling more explicit: `"defaultTraits"`.

Reply:
Ah good point I gotta change the backing `String`. I do prefer `default` though. The `traits` are implied since this is part of a set of traits in the end and this would feel redundant to me.